introduction.md

```diff
@@ -7,3 +7,4 @@ Deadly Skills
 *HTML5
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?